source/jquery.flot.navigate.js

```diff
@@ -107,7 +115,8 @@ can set the default in the options.
             interactive: false,
             active: false,
             cursor: "move",
-            frameRate: 60
+            frameRate: 60,
+            mode: 'smart'
```

Comment:
Why are you changing the default? If you change the default, then you have to explicitly support the non-smart mode. I see no valid reason for changing defaults.

Reply:
Smart-pan is the default behavior for mouse interaction, while normal-pan in the default behavior for touch interaction.

Reply:
BTW, I suppose all of the three modes (normal-pan, smart-pan, and smart-lock-pan) will be supported for either mouse interaction or touch interaction after this change.

Reply:
Again, I don't think the default should be 'smart'. I still haven't been given a reason why we are trying to change the default. Just because your use-case would like to always use the 'smart' mode does not mean the default should change. To be clear, this is a breaking change, behaviorally, for no real good reason (from what I can tell).

To reiterate another point, if the default _were_ to be 'smart', then we would need to have an explicit option for the 'normal' mode. As in, a documented string to use like 'smart' and 'smartLock'. Otherwise, a user will just have to be aware that any garbage they enter for the 'mode' property will be analogous to the 'normal' mode. This is not a good API.

This is another reason to not change the default. We simply document the non-default behaviors of 'smart' and 'smartLock'.